apps/trace/views.py

```diff
@@ -49,11 +53,30 @@ class TraceDetailView(LoginAndTeamRequiredMixin, DetailView, PermissionRequiredM
     permission_required = "trace.view_trace"
 
     def get_queryset(self):
-        return Trace.objects.select_related(
-            "experiment", "session", "participant", "input_message", "output_message"
-        ).filter(team=self.request.team)
+        return (
+            Trace.objects.select_related("experiment", "session", "participant", "input_message", "output_message")
+            .prefetch_related("spans")
```

Comment:
is this prefetch necessary if we're doing a separate query in `get_context_data` to fetch the spans?

Reply:
Thanks for catching this. Not necessary anymore